configs/i90_config.py

```
@@ -256,6 +283,44 @@ def get_sheets_of_interest(self) -> Tuple[List[str], List[str], List[str]]:
 
+    @classmethod
+    def has_volumenes_sheets(cls) -> bool:
+        """
+        Class method to check if this config class has volumenes sheets.
+        Default implementation tries to instantiate the class.
+        Subclasses that require parameters should override this method.
+        """
+        try:
+            # Default behavior: try to instantiate and check
+            instance = cls()
+            return hasattr(instance, 'volumenes_sheets') and instance.volumenes_sheets and any(instance.volumenes_sheets)
```

Pylint (3.3.7)

[error] Instance of 'I90Config' has no 'volumenes_sheets' member; maybe 'volumenes_sheet'?

(E1101)